cobalt/site/docs/development/setup-raspi.md

```diff
@@ -112,8 +112,8 @@ Raspberry Pi.
 
     ```
     $ ssh pi@$RASPI_ADDR
-    $ cd raspi-2_debug
-    $ ./cobalt
+    $ cd raspi-2_devel/install/cobalt_loader
+    $ ./cobalt_loader
```

Comment:
The raspi-2 launcher still runs cobalt using `./cobalt`. Should that be updated?

Reply:
weird, i thought i posted it 
https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/raspi/shared/launcher.py;l=146

the launcher is already checking for `cobalt_loader`